src/crimson/osd/osd_operations/peering_event.cc

```diff
@@ -62,13 +62,13 @@ seastar::future<> PeeringEvent::start()
   return maybe_delay.then([this] {
     return get_pg();
   }).then([this](Ref<PG> pg) {
+    if (!pg) {
+      logger().warn("{}: pg absent, did not create", *this);
+      on_pg_absent();
+      handle.exit();
+      return complete_rctx(pg);
+    }
```

Comment:
the following still  call
       pg->do_peering_event(evt, ctx);
        handle.exit();
        return complete_rctx(pg); 
if not interruptible here, is there any problem?